servicepulse/all-messages.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Audited/Failed Message Display and Discovery
+summary: Describes how ServicePulse displays and allows filtering for audited and failed messages
+component: ServicePulse
+reviewed: 2025-05-11
+related:
+- servicepulse/intro-failed-messages
+- servicepulse/message-details
+---
+
+## Overview
+
+ServicePulse provides visibility into the flow of messages through the system, including both audited and failed messages. It displays the message status, message type, message ID, processing time, critical time, time sent, and time delivered in a comprehensive list view. Clicking any message navigates to a detailed view of that message.
+
+![All Messages](images/all-messages.png 'width=800')
+
+> [!NOTE]
+> Currently, ServicePulse can be connected to only one ServiceControl instance at a time. To view the message list for other ServiceControl instances, go to the [connections in the configuration tab](/servicepulse/host-config.md#configuring-connections-via-the-servicepulse-ui) to set up the URL.
+
+## Message Information
+
+![All Message Info](images/all-messages-info.png 'width=800')
+
+Each row in the list represents a message in the system. Successfully processed messages are prefixed with <img src="images/success-message-icon.png" width="20" alt="successful message">, and failed messages are prefixed with <img src="images/failed-message-icon.png" width="20" alt="failed message">. The following time-related information about the message is also displayed:
```

Comment:
There are 6 different icons so I am not sure if we should be displaying just those two?

Reply:
Included the other icons